pytest_pootle/utils.py

```diff
@@ -40,6 +40,21 @@
 """
 
 
+def setup_store(pootle_path):
+    from pootle_translationproject.models import TranslationProject
+
+    from .factories import StoreDBFactory
+
+    parts = pootle_path.strip("/").split("/")
+    tp = TranslationProject.objects.get(
+        language__code=parts[0], project__code=parts[1])
+    directory = tp.directory
+    for part in parts[2:-1]:
+        directory = directory.child_dirs.get(name=part)
```

Comment:
Don't you find this a complicated way to retrieve the last directory in the path?